students/Pirouz_N/lesson04/mailroom.py

```diff
@@ -56,11 +55,22 @@ def main_menu():
         try:
             option_dict[option]()
         except KeyError:
-            print('Please only type 1 or 2 or 3 or 4 and hit enter. Any other option will not be acceptable.')
+            print(f"Please choose from following options: {str([int(item) for item in list(option_dict)])[1:-1]}")
```

Comment:
what you have is fine - if you're curious on a more pythonic version:
```
print(f"Please choose from following options: {' ,'.join(map(str, option_dict))}")
```

`join()` will join a sequence on an str char (in this case comma); it does require to be a string so I am using map() to match each seq item to a string.

Reply:
Cool! Thank you. I will update it!